schema/property.go

```diff
@@ -83,36 +88,64 @@ func NewPropertyFromObj(id string, rawTypeData interface{}, required bool) *Prop
 	relationProperty, _ := typeData["relation_property"].(string)
 	unique, _ := typeData["unique"].(bool)
 	cascade, _ := typeData["on_delete_cascade"].(bool)
-	properties, _ := typeData["properties"].(map[string]interface{})
 	defaultValue, _ := typeData["default"]
 	if !required && defaultValue == nil {
 		nullable = true
 	}
 	sqlType, _ := typeData["sql"].(string)
 	indexed, _ := typeData["indexed"].(bool)
-	Property := NewProperty(id, title, description, typeID, format, relation, relationColumn, relationProperty,
-		sqlType, unique, nullable, cascade, properties, defaultValue, indexed)
-	return &Property
+
+	var items *Property
+	if itemsRaw, hasItems := typeData["items"]; hasItems {
```

Comment:
It looks `typeID == "array"` check is also required.

Reply:
Done